crate/operator/cratedb.py

```diff
@@ -164,3 +166,22 @@ async def is_cluster_healthy(
             except ProgrammingError as e:
                 logger.warning("Failed to run health check query", exc_info=e)
                 return False
+
+
+async def are_snapshots_in_progress(
+    conn_factory, logger: logging.Logger
+) -> Tuple[bool, Optional[str]]:
```

Comment:
Just a note: personally, i would document what the return values are about. especially the second is not obvious.